src/grimp/adaptors/graph.py

```diff
@@ -102,6 +113,21 @@ def get_import_details(self, *, importer: str, imported: str) -> List[DetailedIm
             imported=imported,
         )
 
+    def find_matching_direct_imports(self, import_expression: str) -> List[Import]:
+        try:
+            importer_expression, imported_expression = import_expression.split(" -> ")
```

Comment:
🐼 Should we split and then strip each side? Do we mind if people do `foo->bar` or `foo     ->    bar`?

Reply:
Good point - but I think I'm okay with being prescriptive about a consistent format, for now at least.